include/nbl/video/IDeviceMemoryBacked.h

```diff
@@ -111,6 +127,7 @@ class IDeviceMemoryBacked : public IBackendObject
         //! members
         SCachedCreationParams m_cachedCreationParams;
         SDeviceMemoryRequirements m_cachedMemoryReqs;
+        void* m_cachedExternalHandle = nullptr;
```

Comment:
what even sets this?

Reply:
see https://github.com/Devsh-Graphics-Programming/Nabla/pull/637#discussion_r1442321812, no need for extra member